### PR TITLE
Corrected twice fields created for university alias and contact details

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Targets #{1042}

Checklist:

- [x] Corrected Space between icon and text written
- [x] Shifted University alias button to down
- [x] Removed twice fields created for university alias
- [x] Removed twice fields created for contact info

### CODE REVIEW

**Comment by srivastava-jyoti** [comment]
In edit.blade.php on inspecting the page in script tag hr.js file is called twice so, when I have commented that it is called only since and the field is showing in once now.

File: Modules/HR/Resources/views/universities/edit.blade.php [hunk 1]
```diff
@@ -52,6 +51,7 @@
         <div id="create_alias_form_list">
            {{-- For inserting dynamic add alias form --}}
         </div>
+        <button type="button" class="btn btn-outline-primary add-aliases mb-3" id="add_contact_btn"><i class="fa fa-plus"></i> Add University Aliases</button>
```
Comment (GauravGusain98) [inline Modules/HR/Resources/views/universities/edit.blade.php:54]: Is this change required Or it is done by mistake?

File: Modules/HR/Resources/views/layouts/master.blade.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 @endsection
 
 @section('js_scripts')
-	<script src="{{ mix('/js/hr.js') }}"></script>
+	{{-- <script src="{{ mix('/js/hr.js') }}"></script> --}}
```
Comment (GauravGusain98) [inline Modules/HR/Resources/views/layouts/master.blade.php:8]: You can remove this line instead of commenting. Leaving commented code is not a good practice.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Some comments. Please check.

File: Modules/HR/Resources/views/universities/edit.blade.php (hunk 1 above)
Comment (GauravGusain98) [reply Modules/HR/Resources/views/universities/edit.blade.php:54]: Cool

**Review by GauravGusain98 — APPROVED** [review]
Looks good
